parquet/src/main/java/org/apache/iceberg/parquet/Parquet.java

```diff
@@ -262,7 +265,7 @@ WriteBuilder withWriterVersion(WriterVersion version) {
             .build();
 
         return new org.apache.iceberg.parquet.ParquetWriter<>(
```

Comment:
Shouldn't we get writer classes now that take a "context" as a creation arg?

Reply:
You mean pass a context to `ParquetWriter`? I am not a huge fan of the introduced context and I consider it very internal to `WriteBuilder` used to create `FileAppender`. I think it is a good idea to avoid leaking it further. Also, such writers have some sort of contexts already. For example, `ParquetProperties` instantiated a few lines above covers some of the properties but not all.